### PR TITLE
Cleanup the code

### PR DESCRIPTION
**Pull request by CaganAgabey**
This pull request cleans the code and changes some functions in some parts with better ones.
I'm not sure if it will work in some parts. I put a comment on these parts, please test them.

### CODE REVIEW

File: background/background.js [hunk 1]
```diff
@@ -106,8 +104,8 @@ chrome.runtime.onInstalled.addListener(details => {
             url: chrome.runtime.getURL("/options/options.html")
         })
     } else if (details.reason == "update" && chrome.runtime.getManifest().version != details.previousVersion) {
-        twpConfig.onReady(function () {
-            if (plataformInfo.isMobile.any) return;
+        twpConfig.onReady(async () => {
```
Comment (FilipePS) [inline background/background.js:107]: Any special reasons to use async here?

File: background/background.js [hunk 2]
```diff
@@ -511,25 +480,25 @@ if (typeof chrome.commands !== "undefined") {
...
-            })
+                }, checkedLastError))
 
             let currentPageTranslatorService = twpConfig.get("pageTranslatorService")
             if (currentPageTranslatorService === "google") {
                 currentPageTranslatorService = "yandex"
-            } else {
+            } else if (currentPageTranslatorService === "yandex") { //I'm not sure in this part
+                currentPageTranslatorService = "bing"
+            } else if (currentPageTranslatorService === "bing") {
```
Comment (FilipePS) [inline background/background.js:501]: This extension has only 2 page translation services. So this doesn't work.

File: background/background.js [hunk 3]
```diff
@@ -581,15 +549,13 @@ if (typeof chrome.commands !== "undefined") {
     })
 }
 
-twpConfig.onReady(function () {
+twpConfig.onReady(async () => {
```
Comment (FilipePS) [inline background/background.js:552]: Any special reasons to use async here?

File: background/background.js [hunk 4]
```diff
@@ -598,7 +564,7 @@ twpConfig.onReady(function () {
     }
 })
 
-twpConfig.onReady(function () {
+twpConfig.onReady(async () => {
```
Comment (FilipePS) [inline background/background.js:567]: Any special reasons to use async here?

File: background/textToSpeech.js [hunk 5]
```diff
@@ -1,114 +1,99 @@
...
+			})
+			
+			return true
+		} else if (request.action === "stopAudio") {
+			audios.forEach(audio => audio.pause())
+			audios = []
+		}
+	})
+	
+	twpConfig.onReady(async () =>
```
Comment (FilipePS) [inline background/textToSpeech.js:93]: Any special reasons to use async here?

**Review by FilipePS — CHANGES_REQUESTED** [review]
Missed editing the `platformInfo` in the `manifest.json` and `chrome_manifest.json` files

File: background/background.js (hunk 2 above)
Comment (CaganAgabey) [reply background/background.js:501]: Then you can make it
`if (currentPageTranslatorService === "google") {
currentPageTranslatorService = "yandex"
} else {
currentPageTranslatorService = "google"
}`

File: background/background.js (hunk 1 above)
Comment (CaganAgabey) [reply background/background.js:107]: This function can work better as asynchronous I think

File: background/background.js (hunk 3 above)
Comment (CaganAgabey) [reply background/background.js:552]: This function doesn't have to work synchronous, so asynchronous can give better performance I think

File: background/background.js (hunk 4 above)
Comment (CaganAgabey) [reply background/background.js:567]: This function includes big processes, so we can make it asynchronous to don't affect overall extension, also nothing blocks it to make asynchronous

File: background/textToSpeech.js (hunk 5 above)
Comment (CaganAgabey) [reply background/textToSpeech.js:93]: It's better to process playback sounds asynchronous I think

**Comment by CaganAgabey** [comment]
Changed manifest.json and chrome_manifest.json too

**Review by FilipePS — COMMENTED** [review]
Thanks for the contribution.
I fixed a few things here:  here https://github.com/FilipePS/Traduzir-paginas-web/commit/5ec537f7a79bd441d66366b5bf8a9f488611b87a

**Pull request merged**